veevavault/1.3/modules/ROOT/pages/veevavault-connector-config-topics.adoc

```diff
@@ -848,7 +264,7 @@ Therefore, the number of records returned per batch would be 10,000 records.
 
 The repeatable streams measure the buffer size in byte measurements. When handling objects, the runtime measures the buffer size using instance counts.
 
-In non-repeatable streams connector operations return streams as the number of records per batch. Repeatable streams return all records at once, so when calculating the in-memory buffer size for repeatable auto-paging, you need to estimate how much memory space each instance takes to avoid running out of memory.
+In non-repeatable streams, connector operations return streams as the number of records per batch. Repeatable streams return all records at once, so when calculating the in-memory buffer size for repeatable auto-paging, you need to estimate how much memory space each instance takes to avoid running out of memory.
```

Comment:
I wonder if we should be referring to repeatable and non-repeatable STREAMING (as a method) rather than STREAMS (as objects)?

Reply:
Will your audience understand what "repeatable auto-paging" is, and by "each instance" do we mean each (non-repeatable) stream?

NOTE that these are all just MY questions, as someone who has less intimate knowledge of this technology than you or your audience. My questions are about term use and definitions, and whether they represent what you mean to say. Feel free to dismiss my questions as you consider them.